client/src/components/cluster.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+import {
+  Flex,
+  Link,
+  Icon,
+  Text,
+  Spacer,
+  HStack,
+  Divider,
+  Button,
+} from "@chakra-ui/react";
+import { FaPlus } from "react-icons/fa";
+import CAvatar from "./clickableAvatar";
+
+const Cluster = ({ data, color }) => {
+  const follow = () => alert("Followed!");
+  const { name, users, defaultCluster } = data;
+  return (
+    <>
+      <Divider />
+      <Flex mt="2vh" mb="1vh" width="full" align="center" direction="row">
```

Comment:
Can you please add a horizontal scroll or left and right navigation arrows like in the [design](https://whimsical.com/fellowcomb-BFUk9i8u4e9jyLcuEivE4h) ?